maven-embedder/src/main/java/org/apache/maven/cli/MavenCli.java

```diff
@@ -444,10 +447,10 @@ private CommandLine cliMerge(CommandLine mavenConfig, CommandLine mavenCli) {
      */
     void logging(CliRequest cliRequest) {
         // LOG LEVEL
-        cliRequest.verbose = cliRequest.commandLine.hasOption(CLIManager.VERBOSE)
-                || cliRequest.commandLine.hasOption(CLIManager.DEBUG);
-        cliRequest.quiet = !cliRequest.verbose && cliRequest.commandLine.hasOption(CLIManager.QUIET);
-        cliRequest.showErrors = cliRequest.verbose || cliRequest.commandLine.hasOption(CLIManager.ERRORS);
+        CommandLine commandLine = cliRequest.commandLine;
+        cliRequest.verbose = commandLine.hasOption(CLIManager.VERBOSE) || commandLine.hasOption(CLIManager.DEBUG);
```

Comment:
This is a nice improvement. Can we move it to a separate PR and apply it basically immediately to 3.9.x and master?

Reply:
Hmm, I think it's a nice improvement, but since it's just formatting related, I would rather just ship it with this PR.

Reply:
Fine